### ISSUE TITLE
Server crash

### ISSUE THREAD

**Issue by Lbniese** [OgarProject/Ogar#444]
Hi,
My server is running on centos 7 digitalocean vps and it's crashing frequently.
I have around 40 people on my server and I haven't touched any files form the repo besides changing the gamemode to experimental.

I'm using this commit: https://github.com/OgarProject/Ogar/commit/e283b3edce31f465e11bbe9629ed83f1c6e5bbc9

Upon crashes I receive this log:

`>TypeError: Cannot read property 'playerTracker' of undefined
    at GameServer.updateClients (/home/ogar/src/GameServer.js:456:24)
    at GameServer.mainLoop (/home/ogar/src/GameServer.js:413:14)
    at wrapper [as _onTimeout] (timers.js:270:14)
    at Timer.listOnTimeout [as ontimeout] (timers.js:121:15)`

**Comment by Luka967** [comment]
What is your gamemode? It might be known to me.

**Comment by Andrews54757** [comment]
I had this problem too but I fixed it (luka967,  told you so in the comments of your teaming update)

**Comment by Andrews54757** [comment]
It is a problem because of the teaming update. to fix,
switch
```
 this.clients[i].playerTracker.update();
this.clients[i].playerTracker.antiTeamTick();
```
with
```
 this.clients[i].playerTracker.antiTeamTick();
        this.clients[i].playerTracker.update();
```

**Comment by Andrews54757** [comment]
(in the function update clients)

**Comment by Luka967** [comment]
weird, it shouldn't throw any error...

**Comment by Andrews54757** [comment]
It does

**Comment by Andrews54757** [comment]
@Luka967 BTW can you make a pull request to revamp the way cells merge in my fork. I tried but it always ended me up with my cells never merging. Or you can just help me. Please?

**Comment by Luka967** [comment]
Just a minute, I already have a ton of stuff to do - install Python, fix bugs at Ogar, get ready to go somewhere soon...

**Comment by Andrews54757** [comment]
Lol

**Comment by Andrews54757** [comment]
I sent a pull request to fix this

**Comment by anderkh** [comment]
The problem with the crash on playerTracker seems to happen after the first users stops playing...  It does seem like swapping the method calls solves it.

**Comment by Andrews54757** [comment]
Also, Ogar doesn't have all the good features. I suggest using another thing like jaralowell/ogarserv or ogarplus/ogarplus or ajs-development/ogar-unlimited or itzlevvie/Ogar-1. (Search them up on google)

**Comment by Luka967** [comment]
@Andrews54757 Working on it!

**Comment by Andrews54757** [comment]
Thanks

**Comment by vram4** [comment]
Fixed in #445
